api/web3server.go

```diff
@@ -233,6 +244,8 @@ func (svr *Web3Server) handlePOSTReq(req *http.Request) interface{} {
 			log.L().Error("web3 server err", zap.String("input", fmt.Sprintf("%+v", web3Req)), zap.Error(err))
 		}
 		web3Resps = append(web3Resps, packAPIResult(res, err, int(web3Req.Get("id").Int())))
+		web3ServerMtc.WithLabelValues(method.(string)).Inc()
+		web3ServerMtc.WithLabelValues("requests_total").Inc()
 	}
 
 	if len(web3Resps) == 1 {
```

Comment:
why need to treat `len() == 1` specially?

Reply:
A json object will be returned as response when len() == 1, whereas a json array returned when len() > 1